src/constants.ts

```diff
@@ -4,9 +4,18 @@ type chainConstants = {
   readonly rpcURL: string;
 };
 
+export const rpcsToRotate = [
```

Comment:
I recommend adding a bunch more rpc's here. 
Then run it with 
`while true; do pnpm envio start; done`
to populate the cache